index.html

```diff
@@ -38,6 +38,193 @@ <h2 class="sub-tittle">I’am a Software Developer</h2>
           </ul> 
         </div>
       </section>
+      <!--  -->
+<section class="second-sec">
+  <h3 class="second-tittle">
+    My Recent Works
+  </h3>
+  <img width="327" height="237" src="./images/Img-Placeholder.png" alt="">
+  <h3 class="second-subtittle">
+    Multi-Post Stories
+  </h3>
+  <p class="second-p">
+    A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+  </p>
+  <div>
+    <ul class="list-btn">
+      <li class="css-btn">css</li>
+      <li class="html-btn">html</li>
+      <li class="boot-btn">bootsrap</li>
+      <li class="ruby-btn">ruby</li>
+    </ul>
+    <button class="btn-top" type="button">See projests</button>
```

Comment:
- [ ] Please fix your see project button and Multi-Post font to match the [figmal design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27)

### **Figmal design**
![seeProButt](https://user-images.githubusercontent.com/55596970/185565442-75891acb-3698-47a5-b55d-da7511796564.png)
### **Your Design**
![client](https://user-images.githubusercontent.com/55596970/185565681-142bf876-f7c1-4918-a14e-82260bcc3ebc.png)